crates/core/tedge/src/cli/certificate/cli.rs

```diff
@@ -405,17 +405,17 @@ pub enum DownloadCertCli {
         )]
         id: String,
 
-        #[clap(long)]
+        #[clap(short = 'p', long = "one-time-password")]
         #[arg(
-            env = "DEVICE_TOKEN",
+            env = "DEVICE_ONE_TIME_PASSWORD",
             hide_env_values = true,
             hide_default_value = true,
             default_value = ""
         )]
-        /// The security token assigned to this device when registered to Cumulocity
+        /// The one-time-password assigned to this device when registered to Cumulocity
```

Comment:
Is really c8y using `one-time-password` instead of `one-time password`? The last hyphen sounds a vert German thing :D When I search in google, I mainly found the usage of `one-time password`
```suggestion
        /// The one-time password assigned to this device when registered to Cumulocity
```